vendor/github.com/opencontainers/runc/libcontainer/system/syscall_linux_32.go

```diff
@@ -0,0 +1,26 @@
+// +build linux
+// +build 386 arm
+
+package system
+
+import (
+	"golang.org/x/sys/unix"
+)
+
+// Setuid sets the uid of the calling thread to the specified uid.
+func Setuid(uid int) (err error) {
```

Comment:
No idea why this file was missing; opened a separate PR for that; https://github.com/containerd/containerd/pull/1889